src/main/java/org/kumoricon/registration/inlinereg/InLineRegistrationController.java

```diff
@@ -57,10 +57,9 @@ public String inLineCheckIn(Model model,
     public String inLineCheckInPost(Model model,
                                     @RequestParam UUID orderUuid,
                                     @AuthenticationPrincipal User user) {
-        log.info("Creating order for in-line registration order {}", orderUuid);
```

Comment:
tried/considered putting a log here so it didnt just go straight into the try-catch, something like `Creating in-line registration order...` or `Attempting to create in-line registration order...` but it got pretty spammy on the console

Reply:
I think that's fine. There's always the trade off between logging what's about to happen in case it fails and logging what did (or didn't) happen